src/pages/HomePage.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import TopSection from '../top_section/top_section.jsx';
+import SectionPhoto from '../section_photo/section_photo.jsx';
+import Header from '../header/header.jsx';
+import Footer from '../footer/footer.jsx';
+import ScrollToTop from '../scroll_to_top.jsx';
+function HomePage() {
+    return (<React.StrictMode>
+        <ScrollToTop />
+        <Header />
+        <TopSection />
+        <SectionPhoto />
+        <Footer />
```

Comment:
`ScrollToTop, Header and Footer` are the same as in `PostPage.jsx` component. you must have a single layout component that will include these 3 components. you can check Andrey's implementation for ideas